src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -15,12 +16,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session as Session);
     const update = appData.update as Update;
 
+    let journeyQuery = "";
+    let journeyType = config.JourneyType.update;
+    if (isRemoveJourney(req)) {
+      journeyType = config.JourneyType.remove;
+      journeyQuery = "?joureny=remove";
```

Comment:
Has this code been unit tested and tested in Tilt/Docker. There's a spelling mistake here which would mean things shouldn't work correctly...

This shouldn't be needed anyway, if you copy other example code in earlier controllers that have been adapted for Remove.

Reply:
Oddly enough it seemed to work when I tested it but have updated and retested it now; reminds me of the mix of redirects some to pages  and some to urls  that might just be a naming mixup otherwise a lot seems to work when it shouldn't.